regolith/tools.py

```diff
@@ -2102,6 +2104,40 @@ def google_cal_auth_flow():
         token.write(creds.to_json())
     # Save the credentials for the next run
 
+def add_to_gitlab(name):
+    """Takes a newly created event, and adds it to the user's google calendar
+
+    Parameters:
+        name - the name/key of the event
+
+    Returns:
+        None
+    """
+
+    tokendir = os.path.expanduser("~/.config/regolith/tokens/gitlab_api")
+    os.makedirs(tokendir, exist_ok=True)
+    tokenfile = os.path.join(tokendir, 'token.json')
+    data = {}
+    # print(tokenfile)
+    if os.path.exists(tokenfile):
+        # if it exists, grab the personal access token
+        with open(tokenfile) as fp:
+            data = json.load(fp)
+
```

Comment:
`else` ?

Think about what behavior the user might want first. If you like, to save time, we can discuss that.